test/functionalTest/cases/1187_empty_attribute_in_create/empty_attribute_in_create.test

```diff
@@ -32,8 +32,8 @@ brokerStart CB
 #
 # 01. POST /v2/entities to create E1/A1=""
 # 02. GET /v2/entities/E1 to see A1==""
-# 03. POST /v2/entities to create E1/A2="", using json object
-# 04. 04. GET /v2/entities/E1 to see A2==""
+# 03. POST /v2/entities to create E1/A2="", using json object (fails, as entity already exists)
```

Comment:
This should be roll-backed... According to specification, POST on /v2/entities/E1 will lead to append/update attributes. Similarly, step 03 should be also rollbacked. Probably, leaving this .test in its original form will work.